app/models/news_article.rb

```diff
@@ -1,3 +1,6 @@
 class NewsArticle < Node
   store_attribute :data, :release_date, Date
-end
\ No newline at end of file
+
+  validates_presence_of :section, :parent
```

Comment:
Yeah I kind of dithered over whether to include this - and decided that I'd leave it up to the developer implementing NewsArticle to remove these validation(s) if they see fit.